### ISSUE TITLE
Pylint silently ignores errors for enabling and disabling messages

### ISSUE THREAD

**Issue by brycepg** [PyCQA/pylint#1964]
### Steps to reproduce
pylint --disable=ALL somefile.py

### Current behavior
The above code will not disable all messages, because `ALL` doesn't do anything. The correct parameter would be `--disable=all`. There is no error and pylint continues to run.

### Expected behavior
Pylint should throw an error when given an unknown message to disable or enable.

Another more insidious example is something like below in `pylintrc`:

```
disable=all
enable=useless-supression,
       unused-import
       no-member
```

The above will silently fail to enable both `unused-import` and `no-member` because there is no comma separating them.

**Comment by PCManticore** [comment]
Sounds good!
